README.md

```diff
@@ -47,11 +47,50 @@ Fb::Page#posts
 Given a page with posts, you can get the posts on the page since creation by calling:
 
 ```ruby
-page = Fb::Page.new access_token: '--valid-access-token--'
+user = Fb::User.new access_token: '--valid-access-token--'
```

Comment:
correct. Then replace with 

```
page = Fb::Page.new id: '--page-id--', access_token: '--valid-access-token--'
```

That's better than introducing the complexity of a `user.pages` association at this point.